examples/iotc_roughtime_client/src/iotc_roughtime_client.c

```diff
@@ -0,0 +1,30 @@
+#include <iotc_roughtime_client.h>
+
+#include "string.h"
+#include <inttypes.h>
+#include <stdint.h>
+#include <stdio.h>
+
+#include "../../common/src/example_utils.h"
+
+/* Application variables. */
+iotc_crypto_key_data_t iotc_connect_private_key_data;
+
+int main() {
+  const char* name = "Cloudflare-Roughtime";
```

Comment:
please revise this `name` variable:
1) name is not descriptive enough. For first read, I don't have a clue whose name is this? Client's name? This roughtime request name? Server name?
2) it's usage path is `iotc_roughtime_getcurrenttime` -> `iotc_roughtime_receive_time` --> `server_name` --> `debug_format`. So based on the usage this is a debug identifier. Is this necessary? If yes, then rename it accordingly.
I might have missed a usage of this variable, if so, please point at it, thx!

Reply:
I think it's only used as a debug identifier. I deleted it!